scripts/test_server.py

```
@@ -0,0 +1,301 @@
+
+    if not dashboard_path.exists() and not platform_path.exists():
+        logger.error("No test HTML files found in web directory")
+        logger.error("Please ensure test-dashboard.html or test-platform.html exist")
+        sys.exit(1)
+
+    # Create and run server
+    server = TestServer(port=args.port, api_url=args.api_url)
+
+    if args.simulate:
+        logger.info("Running in simulation mode - no real cameras needed")
+        simulator = StreamSimulator()
```

Ruff (0.11.9)

Local variable `simulator` is assigned to but never used

Remove assignment to unused variable `simulator`

(F841)